<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Command line suite" duration-ms="15" started-at="2017-09-22T17:23:38Z" finished-at="2017-09-22T17:23:38Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="15" started-at="2017-09-22T17:23:38Z" finished-at="2017-09-22T17:23:38Z">
      <class name="second.NewTest">
        <test-method status="PASS" signature="f()[pri:0, instance:second.NewTest@b684286]" name="f" duration-ms="0" started-at="2017-09-22T13:23:38Z" finished-at="2017-09-22T13:23:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:second.NewTest@b684286]" name="afterMethod" is-config="true" duration-ms="0" started-at="2017-09-22T13:23:38Z" finished-at="2017-09-22T13:23:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="f2()[pri:0, instance:second.NewTest@b684286]" name="f2" duration-ms="0" started-at="2017-09-22T13:23:38Z" finished-at="2017-09-22T13:23:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f2 -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:second.NewTest@b684286]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2017-09-22T13:23:38Z" finished-at="2017-09-22T13:23:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:second.NewTest@b684286]" name="afterMethod" is-config="true" duration-ms="0" started-at="2017-09-22T13:23:38Z" finished-at="2017-09-22T13:23:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:second.NewTest@b684286]" name="afterClass" is-config="true" duration-ms="0" started-at="2017-09-22T13:23:38Z" finished-at="2017-09-22T13:23:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:second.NewTest@b684286]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2017-09-22T13:23:38Z" finished-at="2017-09-22T13:23:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
      </class> <!-- second.NewTest -->
      <class name="second.NewTestTest">
        <test-method status="FAIL" signature="f()[pri:0, instance:second.NewTestTest@7c75222b]" name="f" duration-ms="0" started-at="2017-09-22T13:23:38Z" finished-at="2017-09-22T13:23:38Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Test not implemented]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Test not implemented
	at second.NewTestTest.f(NewTestTest.java:9)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f -->
      </class> <!-- second.NewTestTest -->
    </test> <!-- Command line test -->
  </suite> <!-- Command line suite -->
</testng-results>
